po/de.po

```diff
@@ -0,0 +1,867 @@
+
+#: src/quick-lint-js/error.h
+msgid "unexpected 'finally' without 'try'"
+msgstr "'finally' ohne zugehöriges 'try'"
+
+#: src/quick-lint-js/error.h
+msgid "a function statement is not allowed as the body of {1:singular}"
+msgstr "Ein Funktionsstatement ist ungültig als Körper von {1:singular}"
+
+#: src/quick-lint-js/error.h
+msgid "generator function '*' belongs after keyword function"
+msgstr "'*'-Zeichen für Generatorfunktionen wird nach dem function-Schlüsselwort erwartet"
+
+#: src/quick-lint-js/error.h
+msgid "generator function '*' belongs before function name"
+msgstr "'*'-Zeichen für Generatorfunktionen wird vor Funktionsnamen erwartet"
+
+#: src/quick-lint-js/error.h
+msgid "'in' disallowed in C-style for loop initializer"
+msgstr "Ungültiges 'in' innerhalb Initialisierung der C-ähnlichen for-Schleife"
+
+#: src/quick-lint-js/error.h
+msgid "indexing requires an expression"
+msgstr "Indizierung erfordert einen Ausdruck"
+
+#: src/quick-lint-js/error.h
+msgid "invalid binding in let statement"
+msgstr "Ungültiges Binding in let-Statement"
+
+#: src/quick-lint-js/error.h
+msgid "invalid expression left of assignment"
+msgstr "Ungültiger Ausdruck linkerseits der Zuweisung"
+
+#: src/quick-lint-js/error.h
+msgid "invalid hex escape sequence: {0}"
+msgstr "Ungültige Hex-Escapesequenz: {0}"
+
+#: src/quick-lint-js/error.h
+msgid "invalid lone literal in object literal"
+msgstr "Vereinzeltes Literal in Objekt-Literal"
+
+#: src/quick-lint-js/error.h
+msgid "invalid function parameter"
+msgstr "Ungültiger Funktionsparameter"
+
+#: src/quick-lint-js/error.h
+msgid ""
+"'.' operator needs a key name; use + to concatenate strings; use [] to "
+"access with a dynamic key"
+msgstr "Der '.'-Operator erfordert einen Schlüsselnamen; + verwenden, um Strings zu verketten; [] verwenden, um auf dynamische Schlüssel zuzugreifen"
+
+#: src/quick-lint-js/error.h
+msgid "invalid UTF-8 sequence"
+msgstr "Ungültige UTF-8 Sequenz"
+
+#: src/quick-lint-js/error.h
+msgid "React/JSX is not yet implemented"
+msgstr "React/JSX ist noch nicht implementiert"
+
+#: src/quick-lint-js/error.h
+msgid "keywords cannot contain escape sequences"
+msgstr "Schlüsselworte dürfen keine Escapesequenzen beinhalten"
+
+#: src/quick-lint-js/error.h
+msgid "label named 'await' not allowed in async function"
+msgstr "Labels namens 'await' sind innerhalb async-Functionen verboten"
+
+#: src/quick-lint-js/error.h
+msgid "legacy octal literal may not be BigInt"
+msgstr "Veraltete Oktalliterale sind in BigInts nicht erlaubt"
+
+#: src/quick-lint-js/error.h
+msgid "legacy octal literals may not contain underscores"
+msgstr "Veraltete Oktalliterale dürfen keine Unterstriche enthalten"
+
+#: src/quick-lint-js/error.h
+msgid "{0} with no bindings"
+msgstr "{0} ohne Binding"
+
+#: src/quick-lint-js/error.h
+msgid "a lexical declaration is not allowed as the body of {1:singular}"
+msgstr "Eine lexikalische Deklaration ist nicht als Körper von {0:plural} erlaubt"
```

Comment:
Must fix: You used the wrong placeholder. It should be `{1:singular}`, not `{0:plural}`.